h5p-bildetema/src/components/Header/Header.tsx

```diff
@@ -73,9 +74,28 @@ export const Header: React.FC<HeaderProps> = ({
     return element;
   };
 
-  const handleChangeLanguage = (newLanguage: Language): void => {
-    changeCurrentLanguage(newLanguage);
-    setUserData({ ...userData, currentLanguage: newLanguage });
+  const getLanguagePath = (language: Language): string => {
+    if (!topicId) return `/${language.code}`;
+
+    const topic = topicsFromDB?.find(el => el.id === topicId);
+    const topicWord = topic?.labelTranslations.get(language.code);
+    if (!topicWord) return `/${language.code}`;
```

Comment:
Should we point to the topic id if we can't find the word?

```suggestion
    if (!topicWord) return `/${language.code}/${topicId}`;
```